pkgs/development/tools/misc/pwninit/default.nix

```
@@ -0,0 +1,31 @@
+{ darwin, fetchFromGitHub, lib, rustPlatform, stdenv, openssl, pkg-config, xz }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "pwninit";
+  version = "3.2.0";
+
+  src = fetchFromGitHub {
+    owner = "io12";
+    repo = "pwninit";
+    rev = version;
+    sha256 = "sha256-XKDYJH2SG3TkwL+FN6rXDap8la07icR0GPFiYcnOHeI=";
+  };
+
+  buildInputs = [
+    (lib.optional stdenv.isDarwin darwin.apple_sdk.frameworks.Security)
+    openssl
+    xz
+  ];
```

```suggestion
  buildInputs = [
    openssl
    xz
  ]
  ++ lib.optionals stdenv.isDarwin Security;
```

Again, do not bring darwin into the scope of the expression. List the dependencies directly.